backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -450,6 +460,8 @@ def prompt_responder(
                     outputs=response["output"],
                     document_id=document_id,
                     is_single_pass_extract=False,
+                    profile_manager_id=profile_manager_id,
+                    context=response["metadata"]["context"],
```

Comment:
This might be prone to key errors. It'll be safer to use `.get()` methods - are we sure we'll get this nested dict?

Reply:
Yes we'll get this nested dict. Still lets use .get() to be on safer side.